utils/ffmpeg_install.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import os
+import subprocess
+
+
+def ffmpeg_install_windows():
+    try:
+        zip = "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-master-latest-win64-gpl.zip"
+        r = requests.get(zip)
+        with open("ffmpeg.zip", "wb") as f:
+            f.write(r.content)
```

Comment:
loading 400 MB file into memory is not good idea

Reply:
Well we don't have another choice unfortunately

Reply:
you can do this in a separate function so it will be immediately garbage collected